packages/utils/src/respond.js

```diff
@@ -23,14 +23,14 @@ export function respond(res, responseBody, statusCode) {
     .send(JSON.stringify(responseBody));
 }
 
-export function respondWithDownload(res, filePath, deleteAfterDownload = false) {
+export function respondWithDownload(res, filePath, deleteAfterDownload = true) {
```

Comment:
Swapped the default so that temp files get automatically deleted after download

Reply:
Good call!